ssl/t1_trce.c

```diff
@@ -773,7 +775,7 @@ static int ssl_print_client_hello(BIO *bio, SSL *ssl, int indent,
     BIO_printf(bio, "cipher_suites (len=%d)\n", (int)len);
     if (msglen < len || len & 1)
         return 0;
-    while (len > 0) {
+    while (len > 1) {
```

Comment:
this change is not really fixing an overrun because of the condition above.
note there is a similar loop in line 1085 `while (xlen > 0) {`

Reply:
I would'nt mind changing the other loop as well to avoid unnecessary head scratching...

Reply:
Indeed, see #5675  for main discussion